### ISSUE TITLE
buildTauriRequestData cannot process FormData

### ISSUE THREAD

**Issue by alexsunday** [persiliao/axios-tauri-api-adapter#13]
```javascript
export function buildTauriRequestData(data) {
    if (data === undefined || data === null) {
        return undefined;
    }
    if (typeof data === 'string') {
        return Body.text(data);
    }

// there seems has a bug here, if data is a FormData instance, typeof data === 'object' 
    else if (typeof data === 'object') {
        return Body.json(data);
    }
    else if (data instanceof FormData) {
        // @ts-ignore
        return Body.form(data);
    }
    return Body.bytes(data);
}
```

**Comment by persiliao** [comment]
Thanks for bringing this up, if you have time to submit a pull request

**Comment by Nicoolandgood** [comment]
The issue is fixed in #19, waiting for approval

**Comment by persiliao** [comment]
> The issue is fixed in #19, waiting for approval该问题已修复在 #19 ，等待审批

thanks
